servers/display_server.cpp

```diff
@@ -47,231 +47,351 @@ DisplayServer::DisplayServerCreate DisplayServer::server_create_functions[Displa
 
 int DisplayServer::server_create_count = 1;
 
+#ifndef DISABLE_DEPRECATED
+
+RID DisplayServer::_get_rid_from_name(const String &p_menu_root) const {
+	NativeMenu *nmenu = NativeMenu::get_singleton();
+	if (p_menu_root == "_main") {
```

Comment:
```suggestion
	ERR_FAIL_NULL_V(nmenu, RID());
	if (p_menu_root == "_main") {
```

Reply:
It's not necessary here, it's a private method that's only called after checking `nmenu`, changed it to take `nmenu` as argument instead of calling `get_singleton` again.